pkg/featureSet/persistentImages/persistentImages.go

```diff
@@ -0,0 +1,45 @@
+package persistentImages
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/ocm-container/pkg/engine"
```

Comment:
```suggestion
	"github.com/openshift/ocm-container/pkg/engine"
	log "github.com/sirupsen/logrus"
```

Related to the debug log comment below...